autoload/EasyMotion.vim

```diff
@@ -181,13 +181,23 @@ function! EasyMotion#WB(visualmode, direction) " {{{
 endfunction " }}}
 function! EasyMotion#WBW(visualmode, direction) " {{{
     let s:current.is_operator = mode(1) ==# 'no' ? 1: 0
-    call s:EasyMotion('\(\(^\|\s\)\@<=\S\|^$\)', a:direction, a:visualmode ? visualmode() : '', 0)
+    " Note: Previous regex for all directions was '\(\(^\|\s\)\@<=\S\|^$\)'
```

Comment:
You can remove these note about previous regex because we use git for version control.